test/agent/test_task_agent.py

```diff
@@ -4,10 +4,11 @@
 from camel.configs import ChatGPTConfig
 from camel.typing import TaskType
 
-assert os.environ.get("OPENAI_API_KEY") is not None, "Missing OPENAI_API_KEY"
-
 
 def test_task_specify_ai_society_agent():
+    assert os.environ.get("OPENAI_API_KEY") is not None, \
+        "Missing OPENAI_API_KEY"
```

Comment:
Let's use `,` to format code instead of `\`
```suggestion
    assert os.environ.get("OPENAI_API_KEY", ) is not None, "Missing OPENAI_API_KEY"
```

Reply:
I do not think you should add commas inside random brackets just to please the linter. I suggest to leave it as backslash and then replace plain assert with self.assertEqual() later.